force-app/main/default/triggers/Count.trigger

```diff
@@ -0,0 +1,70 @@
+trigger Count on Task (after insert, after update, after delete) {
```

Comment:
As a best practice, there is usually no business logic in triggers. Refactor into an extra class, please.

Reply:
Trigger Handler Pattern would be worth a search, too. just to get an idea where this can lead to